install_scripts/templates/common/kubernetes-upgrade.sh

```diff
@@ -205,6 +205,18 @@ upgradeKubernetes() {
             airgapLoadReplicatedAddonImagesSecondary
         fi
         kubeadm config migrate --old-config /opt/replicated/kubeadm.conf --new-config /opt/replicated/kubeadm.conf
+
+        if [ "$AIRGAP" != "1" ]; then
+            docker pull "{{ images.kube_apiserver_v1153.name }}"
+            docker pull "{{ images.kube_controller_manager_v1153.name }}"
+            docker pull "{{ images.kube_scheduler_v1153.name }}"
+            docker pull "{{ images.kube_proxy_v1153.name }}"
+        fi
+        docker tag "{{ images.kube_apiserver_v1153.name }}" replicated/kube-apiserver:v1.15.3
```

Comment:
Is the plan to keep the current version of `replicated/kube-apiserver:v1.15.3` around as a backup in case the image is deleted from an installation or to delete that from dockerhub?

Reply:
i dont think we cant delete it since we need to support old installations